registry.yaml

```diff
@@ -215,6 +215,17 @@ packages:
   description: A task runner / simpler Make alternative written in Go
   files:
   - name: task
+- name: gohugoio/hugo
+  type: github_release
+  repo_owner: gohugoio
+  repo_name: hugo
+  asset: 'hugo_{{trimV .Version}}_{{.OS}}-{{.Arch}}.tar.gz'
+  link: https://gohugo.io
+  description: The world’s fastest framework for building websites
+  files:
+  - name: hugo
+    replacements:
+      darwin: macOS
```

Comment:
`replacements` is not here.
https://github.com/suzuki-shunsuke/aqua-registry/blob/fde50932e0d9d55ec1941194a6a1f45f765cdc2a/registry.yaml#L7-L13